test/e2e/infinispan/smoke_test.go

```diff
@@ -88,23 +88,3 @@ func verifyDefaultAuthention(require *require.Assertions, ispn *v1.Infinispan) {
 
 	testAuthentication(ispn, schema, user, pass)
 }
-
-func TestCacheService(t *testing.T) {
```

Comment:
We should update this test to ensure that WellFormed=False is set. As webhooks are not in place when executing this test, it should still be possible to configure the service type.